src/stdlib_linalg_blas_aux.fypp

```diff
@@ -28,9 +28,9 @@ module stdlib_linalg_blas_aux
 
      contains
 
-     !> DCABS1: computes |Re(.)| + |Im(.)| of a double complex number
 
      pure real(dp) function stdlib_dcabs1(z)
+     !> DCABS1 computes |Re(.)| + |Im(.)| of a double complex number
```

Comment:
I think it should be `!!` instead of `!>` [when the docstrings are inside blocks](https://forddocs.readthedocs.io/en/latest/user_guide/writing_documentation.html) (at least it is how it was done in other files). `!>` should be used for pre-documentation.

```suggestion
     !! DCABS1 computes |Re(.)| + |Im(.)| of a double complex number
```

Reply:
Thanks @jvdp1, completed in [5735af9](https://github.com/fortran-lang/stdlib/pull/786/commits/5735af92cdf9aeb683a9b7939f487138cf33be4e)